grammar.js

```diff
@@ -1241,11 +1290,79 @@ module.exports = grammar({
       ),
     ),
 
+    alter_type: $ => seq(
+      $.keyword_alter,
+      $.keyword_type,
+      $.identifier,
+      choice(
+        $.change_ownership,
+        $.set_schema,
+        $.rename_object,
+        seq(
+          $.keyword_rename,
+          $.keyword_attribute,
+          $.identifier,
+          $.keyword_to,
+          $.identifier,
+          optional($._cascade_restrict)
+        ),
+        seq(
+          $.keyword_add,
+          $.keyword_value,
+          optional($._if_not_exists),
+          $.identifier,
+          optional(
+            seq(
+              choice($.keyword_before, $.keyword_after),
+              $.identifier
+            )
+          ),
+        ),
+        seq(
+          $.keyword_rename,
+          $.keyword_value,
+          $.identifier,
+          $.keyword_to,
+          $.identifier,
```

Comment:
```suggestion
          $.literal,
          $.keyword_to,
          $.literal,
```
same, this is modifying enum values not composite type attributes